fcrepo-audit-triplestore/README.md

```diff
@@ -4,34 +4,13 @@ This application implements a bridge to an external, triplestore-based
 [Audit Service](https://wiki.duraspace.org/display/FF/Design+-+Audit+Service)
```

Comment:
If I'm being totally honest, I'm not really clear on what the audit service does. Is it recording a log of every API operation that takes place in Fedora? I can figure it out from reading the processors, but it would be nice if the README at least briefly described what the service did beyond being a "bridge to an external audit service".

Also, some outdated reference here and on the next line
`[Audit Service](https://wiki.duraspace.org/display/FF/Design+-+Audit+Service)` => `[Audit Service](https://wiki.lyrasis.org/display/FF/Design+-+Audit+Service)`
`[Fedora4](http://fcrepo.org)` => `[Fedora](https://duraspace.org/fedora/)`

Reply:
Good point all around here.